docker/config/alembic.ini

```diff
@@ -11,8 +11,6 @@ file_template = %%(rev)s_%%(slug)s
 # the 'revision' command, regardless of autogenerate
 # revision_environment = false
 
-sqlalchemy.url = postgresql://postgres:aPassword@postgresql:5432/socorro_migration_test
```

Comment:
We can take this out because our `alembic/env.py` pulls this value from the environment.